src/app/clusters/messaging-client/messaging-client-cli.c

```diff
@@ -31,45 +31,45 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-/***************************************************************************//**
- * @file
- * @brief CLI for the Messaging Client plugin.
- *******************************************************************************
-   ******************************************************************************/
+/***************************************************************************/ /**
+                                                                               * @file
```

Comment:
This restyle is just wrong.


Maybe we can somehow exclude autogenerated files. Other restyles are reasonable.

Reply:
Fwiw, putting a newline before the start of that second comment makes things look fine.  That's what we've generally done with the generated files to avoid this problem...